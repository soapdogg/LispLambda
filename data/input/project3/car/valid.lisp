(CAR (CONS 2 45))
(CAR (CONS (CAR (CONS 56 43)) T))
(PLUS (CAR (CONS 2 0)) (CAR (CONS 9 3)))