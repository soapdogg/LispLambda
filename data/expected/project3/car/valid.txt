2
56
11